app/views/sessions/new.html.erb

```diff
@@ -0,0 +1,16 @@
+<h1><%= t ".login" %></h1>
+
+<div class="row">
+  <div class="col-md-6 col-md-offset-3">
+    <%= form_for(:session, url: login_path) do |f| %>
```

Comment:
bỏ ()

Reply:
file này em xóa rồi, vì Tân báo làm phần log in rồi